build.gradle

```diff
@@ -29,30 +29,20 @@ sourceCompatibility = '1.7'
 targetCompatibility = '1.7'
 
 
-def cq_version = "5.6.2"
-def jackrabbit_version = "2.6.2"
-
 dependencies {
-    compile "javax.jcr:jcr:2.0"
-    compile "org.apache.jackrabbit:jackrabbit-api:${jackrabbit_version}"
-    compile "org.apache.jackrabbit:jackrabbit-jcr-commons:${jackrabbit_version}"
-    compile "org.apache.sling:org.apache.sling.api:2.4.3-R1488084"
-    compile "org.apache.sling:org.apache.sling.jcr.api:2.1.0"
-    compile "org.apache.sling:org.apache.sling.jcr.resource:2.2.8"
-    compile "com.day.commons:day-commons-text:1.1.8"
-    compile "com.day.cq.wcm:cq-wcm-api:${cq_version}"
-    compile "com.day.cq:cq-commons:${cq_version}", {
-        exclude group: 'org.apache.commons', module: 'commons-email'
+
+    // AEM's new "uber-jar"
+    compile "com.adobe.aem:aem-api:6.0.0.1", {
```

Comment:
What's the benefit of adding the complexity of the additional level of indirection? Are the versions used in multiple places? Is there any duplication? Otherwise it's better to keep the version at the place where it is used instead of mixing it with other concerns (like general project properties).